### PR TITLE
resolving DOCS-279 and DOCS-171

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

**Review by MichaelBaj — DISMISSED** [review]

File: docs/intro_upgrading.md [hunk 1]
```diff
@@ -71,23 +94,25 @@ As an administrator-level user, log into the conductor's PCLI.
...
+The Automated Provisioner upgrades both nodes in a high availability router in series to minimize/avoid downtime. Despite this, it is still recommended to perform upgrade activity during periods of low traffic or maintenance windows.
 
 ## Upgrading using the Interactive Installer
+The Interactive Installer can be used to upgrade a conductor or a router.
 
 1. Launch a Linux command prompt window on the node you wish to upgrade.
 
    :::note
-   To avoid SSH session timeout during installation, it is strongly recommended to use the Screen utility when performing an upgrade in interactive mode.
+   If you are running an older version of the Installer (prior to version 2.7.0), it is strongly recommended to use the Screen utility when performing an upgrade in interactive mode to avoid SSH session timeout. Installer versions 2.7.0 and above do not need to use the Screen utility.
```
Comment (ptimmons) [inline docs/intro_upgrading.md:115]: First choice should be to upgrade the installer if at all possible. If that is impossible, then `screen` is the best path for versions < 2.7.

Comment (ptimmons) [reply docs/intro_upgrading.md:115]: "dnf update 128T-installer"

**Review by ptimmons — CHANGES_REQUESTED** [review]
Minor suggestions.

File: docs/intro_upgrading.md (hunk 1 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/intro_upgrading.md]: added both comments.

**Review by MichaelBaj — APPROVED** [review]

**Review by ptimmons — APPROVED** [review]

**Pull request merged**